app/assets/javascripts/components/CreateTasks.jsx

```diff
@@ -0,0 +1,88 @@
+
+class CreateTasks extends React.Component {
+
+    constructor(props) {
+        super();
+        this.state = {
+            users: [],
+            tasks: []
+        }
+    }
+
+    componentDidMount() {
+        Requester.get('/api/tasks/show?client_id=' + this.props.clientId).then((data) => {
+            this.setState({tasks: data});
+        });
+
+        Requester.get('/api/users/show').then((data) => {
+            this.setState({users: data});
+        });
+    }
+
+    assignTask(taskId, selectId) {
+        select = document.getElementById(selectId);
+        userId = select.value;
+
+        payload = {
+            "user_id": userId,
+            "task_id": taskId,
+            "token": this._getToken()
+        };
+
+        Requester.post('/api/tasks/assign', payload).then((data) => {
+            // Success.
+        });
+    }
+
+    createTask(textId) {
+        textInput = document.getElementById(textId);
+        desc = textInput.value;
```

Comment:
same as the comment about the `select` variable above. Talk to me about this please.